Stripe/STPPaymentCardTextField.m

```diff
@@ -443,9 +600,9 @@ - (void)setCardParams:(STPCardParams *)cardParams {
         [self setText:@"" inField:STPCardFieldTypeExpiration];
```

Comment:
I think removing it would be a breaking API change and I'm not sure what the full consequences are. It is not wholly constructed from other data in this view, it can also save and return non-rendered data. Eg look at `testSetCard_addressFields` in `STPPaymentCardTextFieldTest.m`